auto_round/inference/backend.py

```diff
@@ -172,6 +187,22 @@ def feature_multiply_checker_group_size(
     requirements=["auto-round>=0.5.1"],
 )
 
+# FP8 static quant
+# Weight: FP8, per-channel, may be extended to per-tensor in future
+# Activation: FP8, per-tensor
+
+BackendInfos["auto_round:torch_fp8_static"] = BackendInfo(
+    device=["xpu", "cuda", "cpu"],
+    packing_format="",
+    sym=[True],
+    dtype=["float32", "float16", "bfloat16"],
+    bits=[8],
+    priority=0,
+    feature_checks=[torch_fp8_static_check],
+    alias=["auto_round", "torch"],
+    requirements=["auto-round>0.6.0"],
```

Comment:
Since this PR modifies the core Transformers inference code, I’d prefer to merge it after the 0.6.1 release.

Reply:
This feature is targeted for the INC 3.6, and several loading-related features depend on it, such as NVFP4/MXFP8/MXFP4, and possibly non-transformers model loading @mengniwang95 .

What’s the release plan for version 0.6.1? Could we create the 0.6.1 RC branch first and merge this PR into `main` as soon as possible?

cc @thuang6 @chensuyue